### ISSUE TITLE
FD file are not removed in /proc when client disconnects 

### ISSUE THREAD

**Issue by hiren-bhuva** [zaphoyd/websocketpp#875]
I am using websocketpp as webrtc server. The server works well. I am able to stream in chrome. 
When I connect from the chrome, at server (runnning on linux machine) some fd files are generated ( /prc/PROCESS_ID/fd ). But, when i close the window in google chrome, I am expecting newly created fd to be removed.
This fd are not being removed.
Please, let me know, What can be the issue.

Thanks in Advance.

**Comment by asdika01** [comment]
I have the same issue.
After I moved to a new server I had to update the websocketpp from 0.4.0 to 0.8.2. Everything working well but the server is reaching the max open files with ~1700 connection while the limit is 20,480.

/proc/1075119/fd# ls | wc -l
7764
# cat /proc/1075119/limits | grep files
Max open files            20480                20480                files

Any idea?

**Comment by erikzenker** [comment]
@asdika01 is it really the same issue? The issue of @hiren-bhuva sound like unclosed sockets, but yours sound like problems with fds limits. Am I right?

**Comment by asdika01** [comment]
@erikzenker No, I have the same issue. My limit is on 20,480, I have around 1500 connections but the fd shows much more than that. Once it reached 20,480, the process is dropping all connection until i'm restarting it.

**Comment by erikzenker** [comment]
Okay, maybe you can construct a minimal reproducer and post your code here?
